libs/dh/admin/feature-permissions/src/lib/overview/dh-admin-permission-overview.component.ts

```diff
@@ -48,15 +50,19 @@ import { getPermissionsWatchQuery } from '../shared/dh-get-permissions-watch-que
     DhEmDashFallbackPipeScam,
     WATT_TABLE,
     DhAdminPermissionDetailComponent,
+    DhSharedUiSearchComponent,
   ],
 })
 export class DhAdminPermissionOverviewComponent implements OnInit, OnDestroy {
-  private getPermissionsQuery = getPermissionsWatchQuery();
-  private subscription!: Subscription;
+  private apollo = inject(Apollo);
+  private destroy$ = new Subject<void>();
+
+  query = getPermissionsWatchQuery(this.apollo, undefined);
```

Comment:
suggestion: the `query` is initialized together with the class, so the `inject(Apollo)` will also be available inside the `getPermissionsWatchQuery` function. Let's move it there instead. The `searchTerm` param is also optional, so it can be skipped as it will be `undefined` by default.